src/main/java/seedu/address/model/lesson/Date.java

```diff
@@ -118,8 +118,13 @@ public Date getPreviousDate(DayOfWeek dayOfWeek) {
         return prevDate;
     }
 
+    //@@author angkoonhwee
```

Comment:
You need to end off your claim, if not the rest of the code until EOF will be considered yours

Reply:
![image](https://user-images.githubusercontent.com/62977276/140726901-b488b95f-96f4-425d-a6f1-9d426c2d3d72.png)

Reply:
oh oh okayy psps